xml/tuning_cgroups.xml

```diff
@@ -221,8 +220,130 @@
    systemd.resource-control</command>.
   </para>
  </sect1>
- <sect1 xml:id="sec-tuning-cgroups-info">
-  <title>For More Information</title>
+ 
+ <sect1 xml:id="sec-tuning-cgroups-tasksmax">
+  <title>Preventing Fork Bombs with TasksMax</title>
+  <para>
+      &systemd; 228 shipped with a <literal>DefaultTasksMax</literal> 
+      limit of 512, which is set in 
+      <filename>/etc/systemd/system.conf</filename>. This limits the
+      number of tasks any &systemd; unit can create at one time to
+      512. Previous versions had no default limit. The goal was to
+      improve security by preventing runaway processes from creating
+      excessive forks, or spawning enough threads to exhaust
+      system resources.
+  </para>
+  <para>
+      However, it soon became apparent that there is not a single
+      default that applies to all use cases. 512 is not low enough
+      to prevent a runaway process from crashing a system, and not
+      high enough for processes that create a lot of threads, such
+      as databases. In &systemd; 234 the default was changed to 15%, 
+      which is 4915 tasks (15% of the kernel limit of 32768, 
+      see <command>cat /proc/sys/kernel/pid_max</command>).
+  </para>
+  <para>
+      &sle; ships with a custom configuration that overrides the upstream
+      default. 
+      <filename>/usr/lib/systemd/system.conf.d/20-suse-defaults.conf</filename>
+      overrides the default limit with the following configuration:
+  </para>
+  <screen>
+[Manager]
+DefaultTasksMax=infinity
+  </screen>
+  <para>
+      When you query the value with <command>systemctl</command> it looks like
+      this:
+  </para>
+  <screen>&prompt.user;systemctl show --property DefaultTasksMax
+DefaultTasksMax=18446744073709551615</screen>
+  <para>
+       This is the same as having no limit. It is not a requirement to change the 
+       default, but setting some limits may help to prevent system crashes
+       from runaway processes.
+   </para>
+   <para>
+       Change the global default in
+       <filename>/etc/systemd/system.conf</filename> by uncommenting the 
+       <literal>DefaultTasksMax</literal> line, and changing the value either
+       to a number of tasks, or a percentage, like the following examples:
+   </para>
```

Comment:
Are you sure this works ?

IIRC drop-in files wherever they're located (could be in /usr or /run or ...) take precedence over their main configuration files wherever it's located. This part is debatable but it's how it's implemented so far so we have to deal with it.

So to change the SUSE default, the drop-in itself must be overridden:

```
cat >/etc/systemd/system.conf.d/50-tasksmax.conf
[Manager]
DefaultTasksMax=128
```

or all SUSE specific defaults should be masked and hence the defaults are restored to their upstream values: 

`ln -s /dev/null /etc/systemd/system.conf.d/20-suse-defaults.conf`